extensions/2.0/Khronos/KHR_materials_iridescence/README.md

```diff
@@ -0,0 +1,429 @@
+The thickness of the thin-film is set to `iridescenceThicknessMaximum` if `iridescenceThicknessTexture` is not given.
+If `iridescenceThicknessTexture` is set, the thickness of the thin-film varies between  `iridescenceThicknessMinimum` and `iridescenceThicknessMaximum` as follows:
+
+```glsl
+thickness = mix(iridescenceThicknessMinimum, iridescenceThicknessMaximum, iridescenceThicknessTexture.g)
+```
+
+Aside from light direction and IOR, the thickness of the thin-film defines the variation in hue.
+This effect is the result of constructive and destructive interferences of certain wavelengths.
+If the the optical path difference between the ray reflected at the thin-film and the ray reflected at the base material is half the wavelength (λ), the resulting 180 degree phase shift is cancelling out the reflected light:
+
+<figure style="padding: 10px;">
+<img width=70% src="./figures/interference.png"/>
+<figcaption><em> </em></figcaption>
+</figure>
+
+With a thin-film thickness near half the wavelength of visible light (380 nm - 750 nm), the effect is most visible.
+By increasing the thin-film thickness, multiples of wavelengths are still causing wave interferences, however, as the optical path distance increases, different rays are mixed in.
+This leads to more pastel colored patterns for increased thickness:
+
+<figure style="padding: 10px;">
+<img src="./figures/thickness-comparison.png"/>
+<figcaption><em>Comparison of different iridescence thickness ranges for a constant iridescence IOR value of 1.3 on a dielectric base material with IOR value of 1.5. </em></figcaption>
+</figure>
+
+The thin-film layer can have a different IOR than the underlying material. With `iridescenceIOR` one can set an IOR value for the thin-film layer independently. The more this value differs from the IOR of the base material, the stronger the iridescence. It also has an effect on the optical path difference as visible here:
+
+<figure style="padding: 10px;">
+<img src="./figures/ior-comparison.png"/>
+<figcaption><em>Comparison of different iridescence IOR values for a thin-film thickness range between 200 nm (bottom) and 800 nm (top) on a dielectric base material with IOR value of 1.5.</em></figcaption>
+</figure>
+
+The iridescence effect is modeled via a microfacet BRDF with a modified Fresnel reflectance term that accounts for inter-reflections as shown in the paper from [Belcour/Barla](#theory-documentation-and-implementations).
+
+## Iridescence BRDF
+
+### Metal Base Material
+
+The metal BRDF in the glTF 2.0 core specification in [B.2.1. Metals](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#metals) will be replaced by the following term:
+
+```
+metal_brdf =
+  iridescent_conductor_layer(
+    iridescence_strength = iridescence,
+    iridescence_thickness = iridescenceThickness,
+    iridescence_ior = iridescenceIOR,
+    outside_ior = 1.0,
+    base_f0 = baseColor,
+    specular_brdf = specular_brdf(
+      α = roughness ^ 2)))
```

Comment:
You caught this during the meetings and I just wanted to add a reminder. We've got an extra parenthesis here 😃.

Reply:
Good catch 😄 Totally forgot about that. It's fixed now. 😉

Reply:
Good catch 😄 Totally forgot about that. It's fixed now. 😉